cmd/network.go

```diff
@@ -0,0 +1,368 @@
+package cmd
+
+import (
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+
+	"github.com/ignite/cli-plugin-network/network"
+	"github.com/ignite/cli-plugin-network/network/networkchain"
+	"github.com/ignite/cli-plugin-network/network/networktypes"
```

Comment:
rename to `github.com/ignite/cli-plugin-network/network/types` ?

Reply:
`github.com/ignite/cli-plugin-network/network/networkchain` -> `github.com/ignite/cli-plugin-network/network/chain`

Reply:
By the way, shouldn't `github.com/ignite/cli/ignite` imports be considered "third party" within the plugin repo? If so they should be grouped differently.

Reply:
It should yes, but my `goimports -local` flags is set to `github.com/ignite/cli` which actually matches both repos :P

Reply:
Fixed here 74e9752fd8fdbb06a91b38f66dd2e08f12ec907d
With additional target in Makefile

Reply:
For the package rename, this can be done afterward I think.